app/extensions/safe/manageAuthNotifications.js

```diff
@@ -63,18 +120,15 @@ export const addAuthNotification = ( authReqData, app, sendAuthDecision, store )
         sendAuthDecision( false, authReqData, reqType );
         clearNotification();
     };
-
-    // const successCall = {
-    //     id:
-    // }
+    const textNode = createAuthRequestElement(authReqData);
     const notificationId = Math.random().toString( 36 );
     const theNotification = {
         // TODO: where should ID actually be applied?
         id: notificationId,
         text,
-        isPrompt: true
+        isPrompt: true,
+        textNode
```

Comment:
I think `textNode` is maybe too specific... It could be a non text node, so maybe we just make it 'node' ? or 'reactNode' ?